Sources/MapboxCoreNavigation/DistanceFormatter.swift

```diff
@@ -1,26 +1,26 @@
 import CoreLocation
 import MapboxDirections
 
-struct RoundingTable {
-    struct Threshold {
-        let maximumDistance: Measurement<UnitLength>
-        let roundingIncrement: Double
-        let maximumFractionDigits: Int
+public struct RoundingTable {
```

Comment:
We’re choosing to leave this API undocumented for now because most developers shouldn’t adjust it; usability would likely be impaired with any other values. Still, we could document it for our internal purposes to avoid confusion, hiding the documentation from the published docset using `:nodoc:`.